account_financial_report/wizard/aged_partner_balance_wizard.py

```diff
@@ -18,6 +18,7 @@ class AgedPartnerBalance(models.TransientModel):
     company_id = fields.Many2one(
         comodel_name='res.company',
         default=lambda self: self.env.user.company_id,
+        required=True,
```

Comment:
What if you want to launch the report in multi-company (company=false)?

Reply:
In multi-company (company=false in the report), you will access only to data without company (thus, zero accounts because accounts always has a company).

Reply:
But that's because it's no considered in the SQL. I'm with @elicoidal that we should allow that.

Reply:
And what behavior would you like to have in company=false in the report?

Reply:
@elicoidal @pedrobaeza @mreficent The reports contains already a lot of data, just think of a general ledger printed on 3 companies, better to use them as company dependant, and from one single company at once...At least for my company, we have more than 200000 account move lines in 1 year...

Reply:
But then there's no possibility of obtaining consolidated data. Any way, the problem with this module is the SQLs. It should be done with ORM, and let record rules to act... Performance is not a problem if you use proper `read_group` clauses.

Reply:
@pedrobaeza Agree, but let's target this one for next Odoo version, since it will take some work to do, and not the PR's target, @elicoidal Can you make a issue about this, multicompany aware, and probably on the wizard a many2many_tags to select which company do you want to print...

Reply:
Yeah, but as everything needs to be redone to be adapted to multi-company, I think the best is to change also this. It's something that worths to invest in. Current code is hell to be debugged / modified.

Reply:
I like the idea of a many2many_tags for the company

Reply:
Side note that I will add in the issue, the report must be multi-currency and properly aggregate the currencies